app/models/member.rb

```diff
@@ -22,9 +22,11 @@ def name=(full_name)
   end
 
   def liquid_data
+    full_name = name
     attributes.merge({
-      name: name,
-      welcome_name: name.blank? ? email : name
+      name: full_name,
```

Comment:
Ok. I was only asking because I thought it peculiar there was a change to a model in a PR that otherwise only touches the front end. I thought that change had been made in another PR.